packages/react-vis/jest.config.js

```diff
@@ -0,0 +1,10 @@
+/*eslint-env node*/
+const path = require('path');
+
+module.exports = {
+  testMatch: ['**/?(*-)+(test|tests).[tj]s?(x)'],
```

Comment:
@chrisirhc @Xiot  For now I put a regex to match our current test files, i.e  `foo-test.js` and `bar-tests.js` 

Many repo use the `.test.` convention, I don't have a strong opinion on this kind of naming, but I think this PR is the opportunity to rename all the test files if you want to

Reply:
Do we want to keep tests the separate top level `tests` folder, or use the side-by-side `__tests__` folder.   Or just the files completely side by side.

I tend to to prefer the top level `test` folder as I find it keeps things neater.

Reply:
I'm used to have the files side by side at it's easier for me to spot files without tests and to navigate between a file and its test. 

But I also work on projects with a top folder and we have the coverage reports to track missing tests so it's not a big deal for me